config/jobs/kubernetes-sigs/descheduler/descheduler-presubmits.yaml

```diff
@@ -0,0 +1,45 @@
+# sigs.k8s.io/descheduler presubmits
+presubmits:
+  kubernetes-sigs/descheduler:
+  - name: pull-descheduler-verify-gofmt
+    annotations:
+      testgrid-dashboards: sig-scheduling
+      testgrid-tab-name: pull-descheduler-verify-gofmt
+    decorate: true
+    path_alias: sigs.k8s.io/descheduler
+    always_run: true
+    spec:
+      containers:
+      - image: golang:1.13
+        command:
+        - make
+        args:
+        - verify-gofmt
```

Comment:
Maybe we should add a blanket `make verify` command to the descheduler makefile (I was surprised to see it doesn't already have one), so we don't have to update test-infra for all the different verify tests. 

I would also prefer to add more of our verify tests to this anyway, and it's pretty common for repos to run an umbrella "verify" test

Reply:
Agree. Can you update the Makefile? I will update the PR accordingly.

Reply:
Something like https://github.com/kubernetes-sigs/descheduler/pull/307